onnxruntime/test/onnx/dataitem_request.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "callables.h"
+#include "TestCaseResult.h"
+#include "core/common/common.h"
+#include "core/platform/env_time.h"
+
+
+class ITestCase;
+struct OrtAllocator;
+
+namespace Ort {
+struct Session;
+}
+
+// Class that allows to run a single TestCase either sync or async.
```

Comment:
Should this comment be moved to where the class is declared?

Reply:
I will remove it


---
In reply to: [488986168](https://github.com/microsoft/onnxruntime/pull/5169#discussion_r488986168) [](ancestors = 488986168)